src/app/pages/signing/signing.component.ts

```diff
@@ -16,29 +15,13 @@ import { Fonts } from '../../shared/interfaces';
  * click on the 'Send' button.
  */
 export class SigningComponent implements OnInit {
-  /** Default values */
-  public signingBoxText = '';
-  public backgroundColor = '#ffffff';
-  public textColor = '#000000';
-  public canSend = false;
-  public charCount = 0;
-  public maxCharCount: number;
-  public charCountColor = '#bbbbbb';
-  public name = '[friend name]';
+  public name = 'your friend\'s';
 
   private uid: string;
   private key: string;
 
-  // font service stuff
-  public currentFont = 'Arial'; // Used to select a font from fontDict
-  public currentFontFamily = 'Arial, sans-serif'; // Used to CSS select the font
-  public currentFontDisplay = 'Arial'; // Shown in the Font Dropdown menu
-  public fontDict: Fonts;
-
-  constructor(private route: ActivatedRoute, private authService: AuthService, private msgIo: MsgIoService,
-              private router: Router) {
-    this.fontDict = FontService.fonts;
-  }
+  constructor(private route: ActivatedRoute, private authService: AuthService, public signService: SigningService,
```

Comment:
Why not `signingService`? `signService` sounds a bit off

Reply:
I agree, that seems weird.

Reply:
Ok i'll change it, it just got a bit long in the html